### ISSUE TITLE
Some systems failed to build for Quicklisp dist

### ISSUE THREAD

**Issue by quicklisp** [alex-gutev/static-dispatch#13]
Building with SBCL 2.1.10 / ASDF 3.3.5 for quicklisp dist creation.

Trying to build commit id 8dd50fa5fab91be0f16c40c2fb72a33b3b635351

*static-dispatch/test* fails to build with the following error:

```
; caught WARNING:
;   * is not permitted as an argument to the FUNCTION type specifier
; 
; caught WARNING:
;   * is not permitted as an argument to the FUNCTION type specifier
...
Unhandled UIOP/LISP-BUILD:COMPILE-FILE-ERROR in thread #<SB-THREAD:THREAD "main thread" RUNNING {1001C08103}>: COMPILE-FILE-ERROR while compiling #<CL-SOURCE-FILE "static-dispatch/test" "test" "internals">

```

[Full log here](http://report.quicklisp.org/2021-11-02/failure-report/static-dispatch.html)

**Comment by alex-gutev** [comment]
I will take a look at it. Is this an urgent fix, meaning will the system be removed from quicklisp, if its not fixed before the next release?

**Comment by quicklisp** [comment]
Yes. And the fix is trivial.

**Comment by alex-gutev** [comment]
Should be fixed in commit c605f24.
